files/metricspusher.sh

```
@@ -18,7 +18,7 @@ if [ -z "${HOPRD_PROMETHEUS_PUSHGATEWAY_URL}" ]; then
 fi
 
 # Install required packages
-apt update && apt install -y curl jq
+apt update && apt install -y curl jq 2>&1 > /dev/null
```

Shellcheck (0.10.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)